src/app/blog/[slug]/page.tsx

```diff
@@ -60,15 +61,24 @@ function createBlogPostStructuredData(
 export default function BlogPost({ params }: BlogPostProps) {
   const { slug } = params
   const data = getBlogPostData(slug)
-  const { title, description, image, content } = data
+  const { title, description, image, content, publishedOn, category } = data
 
   return (
     <PageLayout>
       <StructuredDataScript
         structuredData={createBlogPostStructuredData(data)}
       />
-      <PageHeader title={title} description={description} image={image} />
-      {content && <MarkdownContent>{content}</MarkdownContent>}
+      <BreadCrumbs />
+      <div className="m-auto max-w-2xl space-y-6">
+        <BlogHeader
```

Comment:
Maybe `BlogPostHeader` - so that's it's a bit more clear it's not the header for the `Blog` page?